de.fu_berlin.inf.dpp.core/test/junit/de/fu_berlin/inf/dpp/versioning/JGitServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package de.fu_berlin.inf.dpp.versioning;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.Status;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class JGitServiceTest {
+
+    public enum RepositoryOwner {
+        Host, Client
+    }
+
+    @Before
+    public void setUp() {
+    }
+
+    public static Git getSettedUpRepo(RepositoryOwner owner)
```

Comment:
Should be private if only used in this class. If you need this method in another test you should think about a test util class. 
The name **get**SettedUpRepo implies that something is **set** before, but this method creates the Set-up. Therefore I would call this method something like **createTestRepoSetup**.